etc/system-contracts/VERSION.md

```diff
@@ -1,22 +1,22 @@
-This directory was copied from https://github.com/matter-labs/era-system-contracts.
+This directory was copied from https://github.com/matter-labs/era-contracts/tree/release-v20-1.4.1-short-term-fee-model/system-contracts.
 
-The current commit: `a00ab9a11643f3a918ed95cdf8a04edff5499d92`.
+The current repository commit: `2dfbc6bac84ecada93cab4a0dea113bc2aceba1c`.
 
 The following directories/files were copied:
 - [bootloader](bootloader)
 - [contracts](contracts)
 - [scripts](scripts)
 - [hardhat.config.ts](hardhat.config.ts)
 - [package.json](package.json)
-- [SystemConfig.json](SystemConfig.json)
-- [yarn.lock](yarn.lock)
```

Comment:
nit: yarn lock I guess also was taken from the repo

Reply:
AFAIR I just generated it, since there is no longer separate `yarn.lock` for the SC